### PR TITLE
copyediting

### PR DESCRIPTION
**Pull request by casperdcl**
- tabulate generic images (part of #267 but doesn't yet fully resolve)
- misc copyediting
  - typos
  - formatting consistency
  - line wrapping consistency
  - minor rewording
  - `terraform fmt`

### CODE REVIEW

File: docs/guides/getting-started.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Begin by [installing Terraform 1.0 or greater](https://learn.hashicorp.com/tutor
 
 In the project root directory:
 
-1. Create a directory named `shared` to store input data and output artifacts.
+1. Create a directory named `shared` to store input data and output artefacts.
```
Comment (0x2b3bfa0) [inline docs/guides/getting-started.md:13]: Whew! I didn't know that one! 😅 🇬🇧

File: docs/resources/task.md [hunk 2]
```diff
@@ -112,15 +103,13 @@ supported by the underlying cloud provider.
 
 ### Generic
 
-The Iterative Provider offers some common machine images which are roughly the same
-for all supported clouds.
+The Iterative Provider offers some common machine images which are roughly the same for all supported clouds.
 
-- `ubuntu` - Official Ubuntu LTS image, currently 20.04.
+- `ubuntu` - [Latest](https://hub.docker.com/_/ubuntu) official [Ubuntu LTS](https://wiki.ubuntu.com/LTS) image.
```
Comment (0x2b3bfa0) [inline docs/resources/task.md:108]:
```suggestion
- `ubuntu` - [Latest](https://hub.docker.com/_/ubuntu) official [Ubuntu LTS](https://wiki.ubuntu.com/LTS) image.
```

Virtual machine images are hardcoded: a new Ubuntu LTS release does **not** imply an immediate bump in our default images.

* Cloud images usually lag behind official LTS releases.
* We aren't going to upgrade them until they are supported by every provider.

File: docs/resources/task.md [hunk 3]
```diff
@@ -15,7 +15,6 @@ resource "iterative_task" "task" {
 
   environment = {GREETING = "Hello, world!"}
   directory   = "${path.root}/shared"
-
```
Comment (0x2b3bfa0) [inline docs/resources/task.md:18]: > * _When multiple arguments with single-line values appear on consecutive lines at the same nesting level, align their equals signs._
>
> * _When both arguments and blocks appear together inside a block body, place all of the arguments together at the top and then place nested blocks below them. Use one blank line to separate the arguments from the blocks._
> 
> * _Use empty lines to separate logical groups of arguments within a block._
>
> [**(Terraform Style Conventions)**](https://www.terraform.io/docs/language/syntax/style.html)

I would keep the blank line, but, if you prefer not to, equal signs should be aligned. If you don't believe me, ask [`terraform fmt`](https://www.terraform.io/docs/cli/commands/fmt.html) for a second opinion. 😉

**Review by 0x2b3bfa0 — CHANGES_REQUESTED** [review]
Looks good to me except for two nitpicks. Not even🧂 will save you from merciless reviews. 😄

File: docs/resources/task.md (hunk 2 above)
Comment (casperdcl) [reply docs/resources/task.md:108]: ah so didn't you mean to suggest:

```suggestion
- `ubuntu` - Official [Ubuntu LTS](https://wiki.ubuntu.com/LTS) image (currently 20.04).
```

File: docs/resources/task.md (hunk 3 above)
Comment (casperdcl) [reply docs/resources/task.md:18]: in that case all the snippets need updating... 9a25d0c

File: docs/resources/task.md (hunk 2 above)
Comment (0x2b3bfa0) [reply docs/resources/task.md]: 🤦🏼 Sorry! The suggestion was broken.

**Review by 0x2b3bfa0 — APPROVED** [review]

File: docs/resources/task.md (hunk 3 above)
Comment (0x2b3bfa0) [reply docs/resources/task.md:18]: 🙈 A good prize for being pedantic. 😄 I recall fixing some of them manually when reviewing #237 though.

**Pull request merged**